nextflow/accession.nf

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Accession variant files and copy to public FTP.
+
+    Inputs:
+            --accession_props       properties files for accessioning
+            --project_accession     project accession
+            --instance_id           instance id to run accessioning
+            --public_dir            directory for files to be made public
+            --logs_dir              logs directory
+    """
+}
+
+params.accession_props = null
+params.project_accession = null
+params.instance_id = null
+params.public_dir = null
+params.logs_dir = null
+// executables
+params.executable = ["bgzip": "bgzip", "tabix": "tabix", "copy_to_ftp": "copy_to_ftp"]
+// java jars
+params.jar = ["accession_pipeline": "accession_pipeline"]
+// help
+params.help = null
+
+// Show help message
+if (params.help) exit 0, helpMessage()
+
+// Test input files
+if (!params.accession_props || !params.project_accession || !params.instance_id) {
+    if (!params.accession_props)    log.warn('Provide an accessions properties file using --accession_props')
+    if (!params.project_accession)  log.warn('Provide a project accession using --project_accession')
+    if (!params.instance_id)        log.warn('Provide an instance id using --instance_id')
+    exit 1, helpMessage()
+}
+
+accession_props = Channel.fromPath(params.accession_props)
+num_props = Channel.fromPath(params.accession_props).count().value
+// watches public dir for the same number of vcf files as there are property files
+accessioned_vcfs = Channel.watchPath(params.public_dir + '/*.vcf').take(num_props)
```

Comment:
That's an interesting technique I did not know existed. 
I'm still a bit worried about other VCF laying around the public directory but we can solve this another way by ensuring that we only process one submission per directory.

Reply:
Just to confirm the behaviour of this, if there are VCFs in the public directory before the pipeline starts, `watchPath` will ignore them (including if they're overwritten).  But if someone adds some random VCFs to the directory while the accessioning is running, this method won't be able to tell the difference.  I'll add a comment about this and we'll revisit in the future.

Reply:
Actually, I wonder if I could filter the channel to only include VCFs that explicitly match the properties files?  Might be a little safer even if we still have to revisit the larger issue.